src/main/java/com/esri/samples/symbology/graphics_overlay_dictionary_renderer_3D/README.md

```diff
@@ -0,0 +1,39 @@
+<h1>Graphics Overlay Dictionary Renderer 3D</h1>
+
+<p>Display mil2525d symbols in 3D.</p>
+
+<p>The dictionary renderer creates these graphics using a local mil2525d style file and a XML file with key-value 
+attributes for each graphic.</p>
+
+<p><img src="GraphicsOverlayDictionaryRenderer3D.png"/></p>
+
+<h2>How it works</h2>
+
+<p>To apply a <code>DictionaryRenderer</code> and display mil2525d graphics in 3D:</p>
+
+<ol>
+    <li>Create a <code>SymbolDicitonary</code> with the "mil2525d" specification type: <code>SymbolDictionary("mil2525d")</code>.</li>
+    <li>Create a <code>DictionaryRenderer</code> with the symbol dictionary.</li>
+    <li>Set the renderer on a graphics overlay with <code>graphicsOverlay.setRenderer(dictionaryRenderer)</code>.</li>
+    <li>Parse an XML file conforming to the specification. It should have key-value pairs to use as attributes for 
+    each graphic:
+        <ul>
+            <li>Use the name of the XML node as the attribute key and the content of the node as the attribute 
+            value.</li>
+            <li>Get the WKID and coordinates from the XML to create the graphic's geometry.</li>
+        </ul>
+    </li>
+    <li>The other attributes will describe the symbology for the graphic.</li>
+    <li>Create the graphic with the geometry and attributes and add it to the graphics overlay.</li>
+</ol>
+
+<h2>Relevant API</h2>
+
+<ul>
+    <li>GraphicsOverlay</li>
+    <li>DictionaryRenderer</li>
+    <li>DictionarySymbolStyle</li>
+</ul>
+
+<h2>Tags</h2>
+<p>Symbology, 3D</p>
```

Comment:
Would be good to mention "Graphics" here too: a user may want to search for graphics and this would be a good example to see in the search returns.